### ISSUE TITLE
Add ability to hide skills when they are accomplished to declutter Overview

### ISSUE THREAD

**Issue by nedredmond** [Khan/career-competencies#33]
Per feedback from @Myranae

**Comment by Myranae** [comment]
Yes, ability to hide skills when accomplished, but also to only show skills that are currently in progress. For example, I like to choose 2-3 skills during annual planning to focus on, so being able to show those skills specifically would let me focus more on them. Then I could open up other skills as I move to those.
